<!DOCTYPE HTML>
<html lang="en">

<head>
    <meta content="text/html; charset=UTF-8" http-equiv="Content-Type">
    <!-- Hi, Jon Here. Please DELETE the two <script> tags below if you use this HTML, otherwise my analytics will track your page -->
    <!-- Global site tag (gtag.js) - Google Analytics -->
    <script async src="https://www.googletagmanager.com/gtag/js?id=UA-7580334-2"></script>
    <script>
        window.dataLayer = window.dataLayer || [];

        function gtag() {
            dataLayer.push(arguments);
        }

        gtag('js', new Date());

        gtag('config', 'UA-7580334-2');
    </script>

    <title>Abdurrahman Abul-Basher</title>

    <meta content="Jon Barron" name="author">
    <meta content="width=device-width, initial-scale=1" name="viewport">

    <link href="stylesheet.css" rel="stylesheet" type="text/css">
    <link href="images/seal_icon.png" rel="icon" type="images/png">
</head>

<body>
<table
        style="width:100%;max-width:800px;border:0px;border-spacing:0px;border-collapse:separate;margin-right:auto;margin-left:auto;">
    <tbody>
    <tr align="center" style="padding:0px">
        <td>
            <a href="images/abdurrahman.jpg" target="_blank">
                <img alt="profile photo" class="hoverZoomLink"
                     src="images/abdurrahman_circle.jpg" style="width:40%;max-width:40%;">
            </a>
        </td>
    </tr>
    <tr align="center" style="padding:0px">
        <td>
            <p style="text-align:center">

                <a href="https://www.linkedin.com/in/meitantei/" target="_blank"><img alt="Linkedin"
                                                                                      href="https://www.linkedin.com/in/meitantei/"
                                                                                      src="images/Linkedin.png"
                                                                                      style="width:3%"></a>&nbsp&nbsp
                <a href="https://scholar.google.com/citations?hl=en&user=0IPfHrwAAAAJ&view_op=list_works&sortby=pubdate"
                   target="_blank"><img alt="Scholar" src="images/Google_Scholar.png" style="width:3%"></a>&nbsp&nbsp
                <a href="https://github.com/arbasher" target="_blank"><img alt="GitHub" src="images/GitHub.png"
                                                                           style="width:3%"></a>&nbsp&nbsp
                <a href="https://twitter.com/meiarbasher" target="_blank"><img alt="Twitter" src="images/Twitter.png"
                                                                               style="width:3%"></a>&nbsp&nbsp
                <a href="mailto:abdurrahman.mohd@gmail.com" target="_blank"><img alt="Email" src="images/email.svg"
                                                                                 style="width:3%"></a>&nbsp&nbsp
                <a href="data/resume.pdf" target="_blank"><img alt="CV" src="images/download.png" style="width:3%"></a>
            </p>
        </td>
    </tr>
    <tr style="padding:0px">
        <td style="padding:0px">
            <table style="width:100%;border:0px;border-spacing:0px;border-collapse:separate;margin-right:auto;margin-left:auto;">
                <tbody>
                <tr style="padding:0px">
                    <td style="padding:2.5%;width:63%;vertical-align:middle">
                        <p style="text-align:center">
                            <name>Abdurrahman Abul-Basher</name>
                        </p>
                        <p style="font-family:Arial;font-size:16px;color:#424242;indent-text: 5em;line-height: 1.5;">
                            I address complex challenges associated with microorganisms and their interactions (w.r.t <a
                                href="https://en.wikipedia.org/wiki/Metabolic_pathway"
                                style="font-family:Arial;font-size:16px;" target="_blank">metabolic pathway</a>) given
                            <a href="https://en.wikipedia.org/wiki/Multiomics" style="font-family:Arial;font-size:16px;"
                               target="_blank">multi-omics</a> data by utilizing vairous machine learning algorithms
                            (e.g. <a href="https://en.wikipedia.org/wiki/Community_structure"
                                     style="font-family:Arial;font-size:16px;" target="_blank">community detection</a>,
                            <a href="https://en.wikipedia.org/wiki/Multi-label_classification"
                               style="font-family:Arial;font-size:16px;" target="_blank">multi-response prediction</a>,
                            ..etc). I obtained my PhD in <a href="https://en.wikipedia.org/wiki/Bioinformatics"
                                                            style="font-family:Arial;font-size:16px;" target="_blank">Bioinformatics</a>
                            (an interdisciplinary field) from the University of British Columbia (<a
                                href="https://www.ubc.ca/" style="font-family:Arial;font-size:16px;"
                                target="_blank">UBC</a>), where I was advised by Dr. Steven J. Hallam (<a
                                href="https://hallam.microbiology.ubc.ca/" style="font-family:Arial;font-size:16px;"
                                target="_blank">Hallam Lab</a>). Previously, I obtained my bachelor's in computer
                            science from King AbdulAziz University (<a href="https://www.kau.edu.sa/home_ENGLISH.aspx"
                                                                       style="font-family:Arial;font-size:16px;"
                                                                       target="_blank">KAU</a>) and masters in
                            information systems security from <a href="https://www.concordia.ca/"
                                                                 style="font-family:Arial;font-size:16px;"
                                                                 target="_blank">Concordia University</a>, where I
                            worked with <a href="http://dmas.lab.mcgill.ca/fung/index.htm"
                                           style="font-family:Arial;font-size:16px;" target="_blank">Dr. Benjamin C. M.
                            Fung</a> in the <a href="http://dmas.lab.mcgill.ca/"
                                               style="font-family:Arial;font-size:16px;" target="_blank">Data Mining and
                            Security (DMaS) Lab</a> to discover topics from chat logs for crime investigation.
                        </p>
                    </td>
                </tr>
                </tbody>
            </table>

            <table align="center" border="0" cellpadding="20" cellspacing="0" width="100%">
                <tbody>
                <tr>
                    <td>
                        <heading style="font-family:Arial;font-size:28px;color:#2E2E2E">selected softwares</heading>
                    </td>
                </tr>
                </tbody>
            </table>

            <table align="center" border="0" cellpadding="0" cellspacing="0" width="95%">
                <tbody>
                <tr>
                    <td style="padding:5px;width:20%;vertical-align:middle">
                        <img alt="mllr" src="images/software_icons/mllr.png" style="width:50%">
                        <br>
                        <a href="https://github.com/hallamlab/mlLGPR" target="_blank">
                            <papertitle>mlLGPR</papertitle>
                        </a>
                    </td>
                    <td style="padding:5px;width:20%;vertical-align:middle">
                        <img alt="pathway2vec" src="images/software_icons/pathway2vec.png" style="width:50%">
                        <br>
                        <a href="https://github.com/hallamlab/pathway2vec" target="_blank">
                            <papertitle>pathway2vec</papertitle>
                        </a>
                    </td>
                    <td style="padding:5px;width:20%;vertical-align:middle">
                        <img alt="triumpf" src="images/software_icons/triumpf.png" style="width:50%">
                        <br>
                        <a href="https://github.com/hallamlab/triUMPF" target="_blank">
                            <papertitle>triUMPF</papertitle>
                        </a>
                    </td>
                    <td style="padding:5px;width:20%;vertical-align:middle">
                        <img alt="leads" src="images/software_icons/leads.png" style="width:50%">
                        <br>
                        <a href="https://github.com/hallamlab/leADS" target="_blank">
                            <papertitle>leADS</papertitle>
                        </a>
                    </td>
                    <td style="padding:5px;width:20%;vertical-align:middle">
                        <img alt="remap" src="images/software_icons/remap.png" style="width:50%">
                        <br>
                        <a href="https://github.com/hallamlab/reMap" target="_blank">
                            <papertitle>reMap</papertitle>
                        </a>
                    </td>
                </tr>

                <tr>
					<td style="padding:5px;width:20%;vertical-align:middle">
                        <img alt="prepBioCyc" src="images/software_icons/prepBioCyc.png" style="width:50%">
                        <br>
                        <a href="https://github.com/hallamlab/prepBioCyc" target="_blank">
                            <papertitle>prepBioCyc</papertitle>
                        </a>
                    </td>
                    <td style="padding:5px;width:20%;vertical-align:middle">
                        <img alt="strasplit" src="images/software_icons/straSplit.png" style="width:50%">
                        <br>
                        <a href="https://github.com/arbasher/straSplit" target="_blank">
                            <papertitle>straSplit</papertitle>
                        </a>
                    </td>
                    <td style="padding:5px;width:20%;vertical-align:middle">
                        <img alt="mlts" src="images/software_icons/mlts.png" style="width:50%">
                        <br>
                        <papertitle>mltS (coming)</papertitle>
                    </td>
                    <td style="padding:5px;width:20%;vertical-align:middle">
                        <img alt="gan2syn" src="images/software_icons/gan2syn.png" style="width:50%">
                        <br>
                        <papertitle>gan2syn (coming)</papertitle>
                    </td>
					<td style="padding:5px;width:20%;vertical-align:middle">
                        <img alt="pride" src="images/software_icons/pride.png" style="width:50%">
                        <br>
                        <papertitle>pride (coming)</papertitle>
                    </td>
                </tr>
                </tbody>
            </table>

            <table align="center" border="0" cellpadding="20" cellspacing="0" width="100%">
                <tbody>
                <tr>
                    <td>
                        <heading style="font-family:Arial;font-size:28px;color:#2E2E2E">education</heading>
                    </td>
                </tr>
                </tbody>
            </table>

            <table align="center" border="0" cellpadding="20" cellspacing="0" width="100%">
                <tbody>
                <tr>
                    <td style="padding:20px;width:25%;vertical-align:middle">
                        <img alt="UBC" src="images/ubc.jpg" style="width:40%">
                    </td>
                    <td style="padding:20px;width:75%;vertical-align:middle; border-bottom:2px solid #D8D8D8;">
                        <a href="https://www.ubc.ca/" target="_blank">
                            <papertitle>The University of British Columbia (UBC)</papertitle>
                        </a>
                        <br>
                        <strong>
                            <a href="https://bcbioinformaticsgrad.ca/" target="_blank">PhD in Bioinformatics</a>
                        </strong>
                        <br>
                        <strong>Thesis: </strong>
                        <a href="https://open.library.ubc.ca/cIRcle/collections/ubctheses/24/items/1.0394748"
                           target="_blank">Machine learning methods for metabolic pathway inference from genomic
                            sequence information</a>
                        <br>
                        Vancouver, Canada, 2020
                    </td>
                </tr>

                <tr>
                    <td style="padding:20px;width:25%;vertical-align:middle">
                        <img alt="Concordia" src="images/concordia.png" style="width:45%">
                    </td>
                    <td style="padding:20px;width:75%;vertical-align:middle; border-bottom:2px solid #D8D8D8;">
                        <a href="https://www.concordia.ca/" target="_blank">
                            <papertitle>Concordia University</papertitle>
                        </a>
                        <br>
                        <strong>
                            <a href="https://www.concordia.ca/ginacody/info-systems-eng.html" target="_blank">Master of
                                Information Systems Security</a>
                        </strong>
                        <br>
                        <strong>Thesis: </strong>
                        <a href="https://spectrum.library.concordia.ca/35925/7/Abdur_Rahman_MASc_Thesis.pdf"
                           target="_blank">Mining chat logs to extract information about authors and topics for crime
                            investigation</a>
                        <br>
                        Montreal, Canada, 2011
                    </td>
                </tr>

                <tr>
                    <td style="padding:20px;width:25%;vertical-align:middle">
                        <img alt="KAU" src="images/kau.png" style="width:50%">
                    </td>
                    <td style="padding:20px;width:75%;vertical-align:middle;">
                        <a href="https://www.kau.edu.sa/home_ENGLISH.aspx" target="_blank">
                            <papertitle>King AbdulAziz University (KAU)</papertitle>
                        </a>
                        <br>
                        <strong>
                            <a href="https://computing.kau.edu.sa/Default-611-EN" target="_blank">BSc in Computer
                                Science</a>
                        </strong>
                        <br>
                        <strong>Thesis: </strong>
                        University courses timetabling system based on genetic algorithm
                        <br>
                        Jeddah, Saudi Arabia, 2008
                    </td>
                </tr>
                </tbody>
            </table>

            <table align="center" border="0" cellpadding="20" cellspacing="0" width="100%">
                <tbody>
                <tr>
                    <td>
                        <heading style="font-family:Arial;font-size:28px;color:#2E2E2E">professional experience
                        </heading>
                    </td>
                </tr>
                </tbody>
            </table>

            <table align="center" border="0" cellpadding="20" cellspacing="0" width="100%">
                <tbody>
                <tr>
                    <td style="padding:20px;width:25%;vertical-align:middle">
                        <img alt="Hallam" src="images/hallam.jpg" style="width:70%">
                    </td>
                    <td style="padding:20px;width:75%;vertical-align:middle; border-bottom:2px solid #D8D8D8;">
                        <a href="https://hallam.microbiology.ubc.ca/" target="_blank">
                            <papertitle>Hallam Lab</papertitle>
                        </a>
                        <br>
                        <strong>
                            Applied machine learning scientist, Vancouver, Canada, 2020 - Current
                        </strong>
                        <br>
                        <strong>Description:</strong>
                        Developing a framework ("mltS") that estimates reliability scores of each member in an ensemble
                        for multi-label classification to optimizing the inference of metabolic pathways from a genomic
                        sequence information.
                        <br>
                    </td>
                </tr>

                <tr>
                    <td style="padding:20px;width:25%;vertical-align:middle">
                        <img alt="UBC" src="images/ubc.jpg" style="width:60%">
                    </td>
                    <td style="padding:20px;width:75%;vertical-align:middle; border-bottom:2px solid #D8D8D8;">
                        <a href="https://www.ubc.ca/" target="_blank">
                            <papertitle>The University of British Columbia (UBC)</papertitle>
                        </a>
                        <br>
                        <strong>
                            PhD research student, Vancouver, Canada, 2016 - 2020
                        </strong>
                        <br>
                        <strong>Description:</strong>
                        Proposed multiple solutions to predicting metabolic pathways from genomic sequence information
                        (over 5000 organismal genomes) at different levels of complexity and completion using supervised
                        (e.g. <a href="https://github.com/hallamlab/leADS" target="_blank">leADS</a> and <a
                            href="https://github.com/hallamlab/mlLGPR" target="_blank">mlLGPR</a>) and unsupervised
                        (e.g. <a href="https://github.com/hallamlab/triUMPF" target="_blank">triUMPF</a> and <a
                            href="https://github.com/hallamlab/pathway2vec" target="_blank">pathway2vec</a>)
                        machine/deep learning algorithms. My work was the first to establish link between the pathway
                        prediction framework with the downstream pathway enrichment analysis method using machine
                        learning approaches. Furthermore, some of these proposed treatments are extensible to
                        closely-associated studies in bioinformatics and machine learning (multi-label classification).
                        <br>
                    </td>
                </tr>

                <tr>
                    <td style="padding:20px;width:25%;vertical-align:middle">
                        <img alt="GSC" src="images/bcgsc.png" style="width:70%">
                    </td>
                    <td style="padding:20px;width:75%;vertical-align:middle;">
                        <a href="https://bcgsc.ca/" target="_blank">
                            <papertitle>Canada's Michael Smith Genome Sciences Centre</papertitle>
                        </a>
                        <br>
                        <strong>
                            Research assistant, Vancouver, Canada, 2013 - 2016
                        </strong>
                        <br>
                        <strong>Description:</strong>
                        Developed two natural language processing tools to summarize and retrieve articles related to
                        patient genomic and mutation profiles from <a href="https://pubmed.ncbi.nlm.nih.gov/about/"
                                                                      target="_blank">PubMed</a> abstracts (~26
                        million): i)- an event extraction (re-adapting <a
                            href="https://bionlp.utu.fi/eventextractionsystem.html" target="_blank">TEES</a>) and ii)- a
                        discourse parsing (based on convolutional neural networks). In addition to that, I collaborated
                        in developing a named entity recognition system and an efficient clustering algorithm to analyze
                        those articles.
                        <br>
                    </td>
                </tr>
                </tbody>
            </table>

            <table align="center" border="0" cellpadding="20" cellspacing="0" width="100%">
                <tbody>
                <tr>
                    <td>
                        <heading style="font-family:Arial;font-size:28px;color:#2E2E2E">selected publications</heading>
                        <p style="font-family:Arial;font-size:16px;color:#424242;indent-text: 5em;line-height: 1.5;">
                            Representative papers are <span class="highlight">highlighted</span>.
                        </p>
                        <span style="font-family:Arial;font-size:20px; color:#424242; indent-text: 5em; background-color:#D4F5CE;">
                        journal
                    </span>
                        <span style="font-family:Arial;font-size:20px; color:#424242; indent-text: 5em; background-color:#F5F79D;">
                        preprint
                    </span>
                        <span style="font-family:Arial;font-size:20px; color:#424242; indent-text: 5em; background-color:#F2CEF5;">
                    under preparation
                    </span>
                        <span style="font-family:Arial;font-size:20px; color:#FFFFFF; indent-text: 5em; background-color:#65397C;">
                    under review
                    </span>
                        <span style="font-family:Arial;font-size:20px; color:#424242; indent-text: 5em; background-color:#8CB3F4;">
                    conference
                    </span>
                    </td>
                </tr>
                </tbody>
            </table>

            <table style="width:100%;margin-right:auto;margin-left:auto;border:0px;border-spacing:0px;border-collapse:separate;">
                <tbody>
                <tr>
                    <td style="padding:20px;width:25%;vertical-align:middle">
                        <img alt="mltS" src="images/mlts.png" style="width:100%">
                    </td>
                    <td style="padding:20px;width:75%;vertical-align:middle; border-bottom:2px solid #D8D8D8; border-right:10px solid #F2CEF5">
                        <a target="_blank">
                            <papertitle>Leveraging multiple (less-trusted) sources to improve metabolic pathway
                                prediction [TO BE ADDED]
                            </papertitle>
                        </a>
                        <br>
                        <strong>Abdurrahman Abul-Basher</strong>,
                        <a href="https://www.linkedin.com/in/julia-anstett-105b6a35/" target="_blank">Julia Anstett</a>,
                        <a href="https://www.linkedin.com/in/ryan-mclaughlin-725b09117/" target="_blank">Ryan J.
                            McLaughlin</a>,
                        <a href="https://www.linkedin.com/in/aditi-nagaraj-195389176/" target="_blank">Aditi N.
                            Nallan</a>, and 
                        <a href="https://www.linkedin.com/in/steven-hallam-58b5538b/?originalSubdomain=ca"
                           target="_blank">Steven J. Hallam</a>
                        <br>
                        <em>Under preparation</em>, 2021
                        <br>
                        <a target="_blank">[TO BE ADDED]</a>
                        <p></p>
                        <p style="font-family:Arial;font-size:14px;color:#2E2E2E;line-height: 1.2;">
                            This paper presents mltS (multi-label learning based on less-trusted sources) which is an
                            ensemble based learning that leverages the idea of estimating memebers reliability scores in
                            an ensemble given a small reference collection dataset. Using mltS, one can the assess the
                            reliability of each model while performing inference in three ways: "meta-predict (mp)",
                            "meta-weight (mw)", and "meta-adaptive (ma)".
                        </p>
                    </td>
                </tr>

                                <tr>
                    <td style="padding:20px;width:25%;vertical-align:middle">
                        <img alt="reMap" src="images/remap.png" style="width:100%">
                    </td>
                    <td style="padding:20px;width:75%;vertical-align:middle; border-bottom:2px solid #D8D8D8; border-right:10px solid #65397C">
                        <a href="https://doi.org/10.1101/2020.08.21.260109" target="_blank">
                            <papertitle>reMap: relabeling multi-label pathway data with bags to enhance predictive
                                performance
                            </papertitle>
                        </a>
                        <br>
                        <strong>Abdurrahman Abul-Basher</strong> and 
                        <a href="https://www.linkedin.com/in/steven-hallam-58b5538b/?originalSubdomain=ca"
                           target="_blank">Steven J. Hallam</a>
                        <br>
                        <em>Under review</em>, 2021
                        <br>
                        <a href="https://github.com/hallamlab/reMap" target="_blank">code</a>
                        <p></p>
                        <p style="font-family:Arial;font-size:14px;color:#2E2E2E;line-height: 1.2;">
                            This paper presents the reMap framework that performs relabeling examples to a different set
                            of labels, characterized as bags, where a bag comprises of correlated pathways.
                        </p>
                    </td>
                </tr>

                <tr>
                    <td style="padding:20px;width:25%;vertical-align:middle">
                        <img alt="leADS" src="images/leads.png" style="width:100%">
                    </td>
                    <td style="padding:20px;width:75%;vertical-align:middle; border-bottom:2px solid #D8D8D8; border-right:10px solid #D4F5CE">
                        <a href="https://doi.org/10.1101/2020.09.14.297424" target="_blank">
                            <papertitle>leADS: improved metabolic pathway inference based on active dataset
                                subsampling
                            </papertitle>
                        </a>
                        <br>
                        <strong>Abdurrahman Abul-Basher</strong>,
                        <a href="https://www.linkedin.com/in/aditi-nagaraj-195389176/" target="_blank">Aditi N.
                            Nallan</a>,
                        <a href="https://www.linkedin.com/in/julia-anstett-105b6a35/" target="_blank">Julia Anstett</a>, and 
                        <a href="https://www.linkedin.com/in/steven-hallam-58b5538b/?originalSubdomain=ca"
                           target="_blank">Steven J. Hallam</a>
                        <br>
                        <em>Under review</em>, 2021
                        <br>
                        <a href="https://github.com/hallamlab/leADS" target="_blank">code</a>
                        <p></p>
                        <p style="font-family:Arial;font-size:14px;color:#2E2E2E;line-height: 1.2;">
                            This paper presents leADS (multi-label learning based on active dataset subsampling) that
                            leverages the idea of subsampling examples from a pool of multi-label data to reduce the
                            negative impact of training loss.
                        </p>
                    </td>
                </tr>

                <tr>
                    <td style="padding:20px;width:25%;vertical-align:middle">
                        <img alt="triUMPF" src="images/triumpf.png" style="width:100%">
                    </td>
                    <td style="padding:20px;width:75%;vertical-align:middle; border-bottom:2px solid #D8D8D8; border-right:10px solid #D4F5CE">
                        <a href="https://doi.org/10.1101/2020.05.27.119826" target="_blank">
                            <papertitle>Metabolic pathway prediction using non-negative matrix factorization with
                                improved precision
                            </papertitle>
                        </a>
                        <br>
                        <strong>Abdurrahman Abul-Basher</strong>,
                        <a href="https://www.linkedin.com/in/ryan-mclaughlin-725b09117/" target="_blank">Ryan J.
                            McLaughlin</a>, and 
                        <a href="https://www.linkedin.com/in/steven-hallam-58b5538b/?originalSubdomain=ca"
                           target="_blank">Steven J. Hallam</a>
                        <br>
                        <u>Short version:</u> <em>International Conference on Computational Advances in Bio and medical
                        Sciences (ICCABS)</em>, 2020
                        <br>
                        <u>Long version:</u> <em>Journal of computational biology (JCB)</em>, 2021
                        <br>
                        <a href="https://github.com/hallamlab/triUMPF" target="_blank">code</a>
                        <p></p>
                        <p style="font-family:Arial;font-size:14px;color:#2E2E2E;line-height: 1.2;">
                            This paper presents triUMPF (triple non-negative matrix factorization (NMF) with community
                            detection for metabolic pathway inference) that combines three stages of NMF to capture
                            myriad relationships between enzymes and pathways within a graph network followed by
                            community detection to extract higher order structure based on the clustering of vertices
                            sharing similar statistical properties.
                        </p>
                    </td>
                </tr>

                <tr>
                    <td style="padding:20px;width:25%;vertical-align:middle">
                        <img alt="pathway2vec" src="images/pathway2vec.png" style="width:100%">
                    </td>
                    <td style="padding:20px;width:75%;vertical-align:middle; border-bottom:2px solid #D8D8D8; border-right:10px solid #D4F5CE">
                        <a href="https://academic.oup.com/bioinformatics/advance-article/doi/10.1093/bioinformatics/btaa906/5932372"
                           target="_blank">
                            <papertitle>Leveraging heterogeneous network embedding for metabolic pathway prediction
                            </papertitle>
                        </a>
                        <br>
                        <strong>Abdurrahman Abul-Basher</strong> and 
                        <a href="https://www.linkedin.com/in/steven-hallam-58b5538b/?originalSubdomain=ca"
                           target="_blank">Steven J. Hallam</a>
                        <br>
                        <em>Bioinformatics</em>, 2020
                        <br>
                        <a href="https://github.com/hallamlab/pathway2vec" target="_blank">code</a>
                        <p></p>
                        <p style="font-family:Arial;font-size:14px;color:#2E2E2E;line-height: 1.2;">
                            This paper presents pathway2vec, a software package consisting of six representational
                            learning based modules used to automatically generate features for pathway inference.
                        </p>
                    </td>
                </tr>

                <tr>
                    <td style="padding:20px;width:25%;vertical-align:middle">
                        <img alt="mlLGPR" src="images/mllr.png" style="width:100%">
                    </td>
                    <td style="padding:20px;width:75%;vertical-align:middle; border-bottom:2px solid #D8D8D8; border-right:10px solid #D4F5CE">
                        <a href="https://journals.plos.org/ploscompbiol/article?id=10.1371/journal.pcbi.1008174"
                           target="_blank">
                            <papertitle>Metabolic pathway inference using multi-label classification with rich pathway
                                features
                            </papertitle>
                        </a>
                        <br>
                        <strong>Abdurrahman Abul-Basher</strong>,
                        <a href="https://www.linkedin.com/in/ryan-mclaughlin-725b09117/" target="_blank">Ryan J.
                            McLaughlin</a>, and 
                        <a href="https://www.linkedin.com/in/steven-hallam-58b5538b/?originalSubdomain=ca"
                           target="_blank">Steven J. Hallam</a>
                        <br>
                        <em>PLOS Computational Biology</em>, 2020
                        <br>
                        <a href="https://github.com/hallamlab/mlLGPR" target="_blank">code</a>
                        <p></p>
                        <p style="font-family:Arial;font-size:14px;color:#2E2E2E;line-height: 1.2;">
                            This paper presents mlLGPR (multi-label logistic regression for pathway prediction)
                            framework that uses supervised multi-label classification and rich pathway features to infer
                            metabolic networks at the individual, population and community levels of organization.
                        </p>
                    </td>
                </tr>

                <tr>
                    <td style="padding:20px;width:25%;vertical-align:middle">
                        <img alt="CBT" src="images/cbt.png" style="width:100%">
                    </td>
                    <td style="padding:20px;width:75%;vertical-align:middle; border-bottom:2px solid #D8D8D8; border-right:10px solid #F5F79D">
                        <a href="data/cbt.pdf" target="_blank">
                            <papertitle>Modeling metabolic pathways as bags (with augmentation)</papertitle>
                        </a>
                        <br>
                        <strong>Abdurrahman Abul-Basher</strong> and 
                        <a href="https://www.linkedin.com/in/steven-hallam-58b5538b/?originalSubdomain=ca"
                           target="_blank">Steven J. Hallam</a>
                        <br>
                        <em>arXiv</em>, 2019
                        <br>
                        <a href="https://github.com/hallamlab/cbt" target="_blank">code</a>
                        <p></p>
                        <p style="font-family:Arial;font-size:14px;color:#2E2E2E;line-height: 1.2;">
                            This paper presents the CBT (correlated bag pathway) package comprising of three
                            hierarchical mixture models: SOAP (sparse correlated bag pathway), SPREAT (distributed
                            sparse correlated bag pathway), and CTM (correlated topic model) to characterize pathways.
                        </p>
                    </td>
                </tr>

                <tr>
                    <td style="padding:20px;width:25%;vertical-align:middle">
                        <img alt="exbl" src="images/exbl.png" style="width:100%">
                    </td>
                    <td style="padding:20px;width:75%;vertical-align:middle; border-bottom:2px solid #D8D8D8; border-right:10px solid #F5F79D">
                        <a href="https://www.biorxiv.org/content/10.1101/034397v1.abstract" target="_blank">
                            <papertitle>Event extraction from biomedical literature</papertitle>
                        </a>
                        <br>
                        <strong>Abdurrahman Abul-Basher</strong>,
                        Alexander S Purdy, and 
                        <a href="https://www.bcgsc.ca/people/inanc-birol" target="_blank">Inanç Birol</a>
                        <br>
                        <em>bioRxiv</em>, 2015
                        <br>
                        <p></p>
                        <p style="font-family:Arial;font-size:14px;color:#2E2E2E;line-height: 1.2;">
                            This paper explores the latest advances in automated event extraction methods in the
                            biomedical domain, focusing primarily on tools participated in the Biomedical NLP (BioNLP)
                            Shared Task (ST) competitions.
                        </p>
                    </td>
                </tr>

                <tr>
                    <td style="padding:20px;width:25%;vertical-align:middle">
                        <img alt="atot" src="images/atot.png" style="width:100%">
                    </td>
                    <td style="padding:20px;width:75%;vertical-align:middle; border-right:10px solid #D4F5CE">
                        <a href="https://link.springer.com/article/10.1007/s10115-013-0617-y" target="_blank">
                            <papertitle>Analyzing topics and authors in chat logs for crime investigation</papertitle>
                        </a>
                        <br>
                        <strong>Abdurrahman Abul-Basher</strong> and 
                        <a href="http://dmas.lab.mcgill.ca/fung/" target="_blank">Benjamin C. M. Fung</a>
                        <br>
                        <em>Knowledge and Information Systems</em>, 2013
                        <br>
                        <a href="https://github.com/arbasher/atot" target="_blank">code</a>
                        <p></p>
                        <p style="font-family:Arial;font-size:14px;color:#2E2E2E;line-height: 1.2;">
                            This paper presents a topic search model to extract topics, compute the contribution of
                            authors in these topics, and study the transitions of these topics over time and
                            characterize authors-topics over time given crime-relevant chat logs.
                        </p>
                    </td>
                </tr>
                </tbody>
            </table>

            <table align="center" border="0" cellpadding="20" cellspacing="0" width="100%">
                <tbody>
                <tr>
                    <td>
                        <heading style="font-family:Arial;font-size:28px;color:#2E2E2E">posters</heading>
                    </td>
                </tr>
                </tbody>
            </table>

            <table align="center" border="0" cellpadding="20" width="100%">
                <tbody>
                <tr>
                    <td style="padding:20px;width:25%;vertical-align:middle;font-family:Arial;font-size:28px;">
                        <img alt="cmde" src="images/cmde.png" style="width:70%">
                    </td>
                    <td style="padding:20px;width:75%;vertical-align:middle;">
                        <a href="data/cmde.pdf" target="_blank">
                            <papertitle>Machine Learning Approach to Recovering Metabolic Pathways from Metagenomics
                                Sequences
                            </papertitle>
                        </a>
                        <br>
                        <strong>Abdurrahman Abul-Basher</strong> and 
                        <a href="https://www.linkedin.com/in/steven-hallam-58b5538b/?originalSubdomain=ca"
                           target="_blank">Steven J. Hallam</a>
                        <br>
                        <em>CMDE</em>, 2016
                    </td>
                </tr>
                </tbody>
            </table>

            <table align="center" border="0" cellpadding="20" cellspacing="0" width="100%">
                <tbody>
                <tr>
                    <td>
                        <heading style="font-family:Arial;font-size:28px;color:#2E2E2E">teachings</heading>
                    </td>
                </tr>
                </tbody>
            </table>

            <table align="center" border="0" cellpadding="20" width="100%">
                <tbody>
                <tr>
                    <td style="padding:20px;width:25%;vertical-align:middle;font-family:Arial;font-size:28px;">
                        <img alt="DSCI" src="images/ubc-mds.png" style="width:70%">
                    </td>
                    <td valign="center" width="75%">
                        <a href="https://masterdatascience.ubc.ca/programs/vancouver" target="_blank">Graduate Student
                            Instructor,
                            DSCI 571 Supervised Learning I, 2016</a>
                        <br>
                        <a href="https://masterdatascience.ubc.ca/programs/vancouver" target="_blank">Graduate Student
                            Instructor,
                            DSCI 573 Feature and Model Selection, 2017</a>
                        <br>
                        <a href="https://masterdatascience.ubc.ca/programs/vancouver" target="_blank">Graduate Student
                            Instructor,
                            DSCI 575 Advanced Machine Learning, 2017</a>
                    </td>
                </tr>
                </tbody>
            </table>

            <table align="center" border="0" cellpadding="20" cellspacing="0" width="100%">
                <tbody>
                <tr>
                    <td>
                        <heading style="font-family:Arial;font-size:28px;color:#2E2E2E">fellowships and awards</heading>
                        <ul style="font-family:Arial;font-size:15px;color:#151515;">
                            <li>Four Year Fellowships (4YF) ($18,200 per year + tuition fee), The University of British
                                Columbia (UBC), Canada. 2013-2017.
                            </li>
                            <li>Faculty of Science - Graduate Support Initiative (GSI) Fund ($8,500 per year), The
                                University of British Columbia (UBC), Canada.2013-2017.
                            </li>
                            <li>Power Corporation of Canada Graduate Fellowships ($5,000), Concordia University, Canada.
                                2009-2010.
                            </li>
                            <li>Concordia Graduate Student Support Program (GSSP) ($15,000 per year), Concordia
                                University, Canada. 2009-2011.
                            </li>
                            <li>First Honor Graduate for graduating with high GPA from King AbdulAziz University, Saudi
                                Arabia. 2008.
                            </li>
                        </ul>
                    </td>
                </tr>
                </tbody>
            </table>

            <table align="center" border="0" cellpadding="20" cellspacing="0" width="100%">
                <tbody>
                <tr>
                    <td>
                        <heading style="font-family:Arial;font-size:28px;color:#2E2E2E">misc</heading>
                        <ul style="font-family:Arial;font-size:15px;color:#151515;">
                            <li>I am quadrilingual: I speak fluent English, Arabic, Bengali and conversational in
                                Hindi-Urdu.
                            </li>
                            <li>In my spare time, I enjoy reading novels, history, and writing poetry in Arabic.</li>
                        </ul>
                    </td>
                </tr>
                </tbody>
            </table>

            <table align="center" border="0" cellpadding="20" cellspacing="0" width="100%">
                <tbody style="display: inline-block; border: 1px #2E2E2E; max-height: 500px; overflow-y: scroll">
                <tr>
                    <td>
                        <heading style="font-family:Arial;font-size:28px;color:#2E2E2E">tweets by meiarbasher</heading>
                        <a class="twitter-timeline" href="https://twitter.com/meiarbasher?ref_src=twsrc%5Etfw"></a>
                        <script async charset="utf-8" src="https://platform.twitter.com/widgets.js"></script>
                    </td>
                </tr>
                </tbody>
            </table>

            <table style="width:100%;border:0px;border-spacing:0px;border-collapse:separate;margin-right:auto;margin-left:auto;">
                <tbody>
                <tr align="center">
                    <td>
                        <p style="text-align:right;font-size:small;">
                            Webpage design courtesy of <a href="https://github.com/jonbarron/jonbarron_website"
                                                          target="_blank">Jon Barron</a>.
                        </p>
                    </td>
                </tr>
                </tbody>
            </table>

            <table style="width:100%;border:0px;border-spacing:0px;border-collapse:separate;margin-right:auto;margin-left:auto;">
                <tbody>
                <tr align="center">
                    <td>
                        <!--                        <a title="free world map tracker" href="https://smallcounter.com/vmap/1596511028/">
                                                <img title="free world map counter" src="https://smallcounter.com/map/view.php?type=180&id=1596511028" border="1" alt="world map hits counter" /></a><br>

                                                <a href="https://smallcounter.com"></a>
                                                <script type="text/javascript" src="https://freehitcounters.org/count/64uf"></script><br>
                                                <a href='https://www.symptoma.ro/'>www.symptoma.ro</a>
                                                <script type='text/javascript' src='https://whomania.com/ctr?id=434a24e4b2451de78a6c1330dfaf5931120650e7'></script>-->

                        <script async="async"
                                src="//rf.revolvermaps.com/0/0/4.js?i=5w2w9clw8gl&amp;m=0&amp;h=128&amp;c=ff0000&amp;r=0"
                                type="text/javascript"></script>
                        <script src='https://whomania.com/ctr?id=434a24e4b2451de78a6c1330dfaf5931120650e7'
                                type='text/javascript'></script>

                        <script src="https://freehitcounters.org/count/64uf" type="text/javascript"></script>
                        <br>
                        <a href='https://www.symptoma.ro/' target="_blank">www.symptoma.ro</a>
                        <script src='https://whomania.com/ctr?id=434a24e4b2451de78a6c1330dfaf5931120650e7'
                                type='text/javascript'></script>
                    </td>
                </tr>
                </tbody>
            </table>

        </td>
    </tr>
</table>
</body>

</html>
